.github/workflows/quest-bulk.yml

```diff
@@ -1,5 +1,7 @@
 name: "bulk quest import"
 on:
+  schedule:
+    - cron: '0 2 * * *' # UTC time, that's 9:00 pm EST, 6:00 pm PST.
```

Comment:
@davidbritch I picked a different time to match likely downtime in the UK. Make a suggestion if you want a different time.